### ISSUE TITLE
Mark archived packages as deprecated

### ISSUE THREAD

**Issue by pfitzseb** [JuliaComputing/JuliaHub-Feedback#43]
Both the package overview and the package detail pages should probably indicate archived packages somehow. We could maybe even penalize them in the package search.

**Comment by nsajko** [comment]
I agree with this issue as stated: "Both the package overview and the package detail pages should probably indicate archived packages somehow. We could maybe even penalize them in the package search."

I guess this issue was replaced with an issue on a private repository?

**Comment by mortenpi** [comment]
I think it was meant to reference #42 actually. I believe this repository was renamed at one point.

**Comment by pfitzseb** [comment]
Yep. I don't think we ever actually got around to implementing the "package is archived" badge though.
